README.md

```diff
@@ -48,12 +59,76 @@ In the European Union, this extension's fields correspond to [eForms BG-704 (Rew
 }
 ```
 
+### Lot
+
+An example with multiple prizes for a single lot.
+
+```json
+{
+  "tender": {
+    "lots": [
+      {
+        "id": "LOT-0001",
+        "designContest": {
+          "selectedParticipants": [
+            {
+              "id": "1",
+              "name": "Brigitte Hermon"
+            },
+            {
+              "id": "2",
+              "name": "Paolo Travino"
+            }
+          ],
+          "hasPrizes": true,
+          "prizes": {
+            "details": [
+              {
+                "id": "1",
+                "description": "A lump sum indemnity of an amount identical to that paid to unsuccessful competitors will be paid to the winning team in the form of an advance on the project management contract at the end of the competition; this sum being credited to the amount of fees to be collected subsequently under the project management contract.",
+                "value": {
+                  "amount": 3000,
```

Comment:
Should this be the same value as the next prize in the `.details` array?

Reply:
I find the description a bit confusing. It mentions an amount paid to unsuccessful competitors, is that reflected in the data anywhere?